### PR TITLE
feat: add `universal`

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [x] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

### CODE REVIEW

File: projects/universal/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+```
+
+Alternatively, if you have a standalone app that is initialized using the bootstrapApplication function, you can import
+`UniversalModule` in the following manner:
+
+```ts
+const serverConfig: ApplicationConfig = {
+  providers: [
+    provideServerRendering(),
+    importProvidersFrom(UniversalModule), // <-- add this
```
Comment (waterplea) [inline projects/universal/README.md:55]: Let's change instructions to use `UNIVERSAL_PROVIDERS`.

File: projects/universal/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+SSR.
+
+> **IMPORTANT:** This library relies on **_Node.js_ v10** and above on your server side
+
+## Mocks
+
+Add following line to your `server.ts` to mock native classes used in other @ng-web-apis packages:
+
+```js
+import '@ng-web-apis/universal/mocks';
```
Comment (waterplea) [inline projects/universal/README.md:26]: Did you add it to current demo?

File: projects/universal/README.md (hunk 1 above)
Comment (splincode) [reply projects/universal/README.md:55]: fixed

File: projects/universal/README.md (hunk 2 above)
Comment (splincode) [reply projects/universal/README.md:26]: https://github.com/ng-web-apis/common/blob/main/apps/demo/server.ts#LL1C1-L1C1

**Pull request merged**